.gitignore

```diff
@@ -7,3 +7,4 @@ example/settings_private.py
 /.tox/
 /htmlcov/
 /docs/_build/
+.eggs/
```

Comment:
I included this in the git ignore to clean up `git status` after you've run tests (stolen from [here](https://github.com/github/gitignore/blob/master/Python.gitignore)).